app/lib/models.py

```diff
@@ -0,0 +1,11 @@
+from app import db
+
+
+class Base(db.Model):
+
```

Comment:
Let's add a small comment here explaining all classes should inherit from this model or that it provides common fields for all models.